src/main/scala/org/opennetworkinsight/FlowPreLDA.scala

```diff
@@ -188,11 +204,9 @@ object FlowPreLDA {
             Array[String]()
           }
 
-          val totalData: RDD[String] = datanoheader.union(sc.parallelize(scoredData))
-
-          val datagood: RDD[String] = totalData.filter(line => line.split(",").length == 27)
+          val totalData: RDD[String] = rawdata.union(sc.parallelize(scoredData))
 
-          val data_with_time = datagood.map(_.trim.split(',')).map(FlowWordCreation.addTime)
+          val data_with_time = totalData.map(_.trim.split(',')).map(FlowWordCreation.addTime)
```

Comment:
again, you did the right thing making a minimal change to get parquet working with the existing code, but it seems weird to load it into to parquet, make a string than have to trim and parse and access things with column indices... need to think of a later touch up for this